server/cli/src/cli.rs

```diff
@@ -631,6 +637,34 @@ fn export_paths(name: &str) -> (PathBuf, PathBuf, PathBuf) {
     (export_folder, export_file_path, users_file_path)
 }
 
+fn run_yarn_install(directory: &str) -> Result<(), Box<dyn std::error::Error>> {
+    let mut convert_dir = directory.to_owned();
+    convert_dir.push_str("/convert_data_js");
+
+    if !Path::new(&convert_dir).exists() {
+        info!(
+            "No conversion function for {}. Skipping esbuild install.",
+            convert_dir
+        );
+        return Ok(());
+    }
+
+    let status = Command::new("yarn")
+        .args(["install", "--cwd", &convert_dir, "--no-lockfile"])
```

Comment:
no lock file because we are using this one package in isolation, so don't need a global version manager (I think!?)